tests/phases/install/quayside_assembly_tow/test_moored.py

```diff
@@ -39,9 +47,18 @@ def test_simulation_setup():
 @pytest.mark.parametrize(
     "weather", (None, test_weather), ids=["no_weather", "test_weather"]
 )
-@pytest.mark.parametrize("config", (config, no_supply))
+@pytest.mark.parametrize(
+    "config",
+    (config, multi_assembly, multi_tow, multi_assembly_multi_tow, no_supply),
+    ids=[
+        "1 assembly, 1 tow",
+        "3 assembly, 1 tow",
+        "1 assembly, 3 tow",
+        "3 assembly, 3 tow",
+        "no supply",
+    ],
+)
 def test_for_complete_logging(weather, config):
```

Comment:
This should also test the number of assembly lines and towing vessels given the parameterization.

Reply:
I believe it does, unless I'm missing something. There are yaml files that set the # of assembly lines or tow vessels and they're assigned to "config". 

[https://github.com/WISDEM/ORBIT/blob/SemiTaut_Mooring_Update/tests/phases/install/quayside_assembly_tow/test_moored.py#L21-L26](url)

Reply:
This is only testing the completion of the installation, but I'd clarify my recommendation to say that we should parameterize the above setup test similarly to check that all of the configurations get set up correctly.